cinema/tests/test_view_api.py

```diff
@@ -0,0 +1,45 @@
+from django.test import TestCase
```

Comment:
This is unnecessary.
Better check, that some methods are not allowed

Reply:
done